packages/coding-standard/tests/Rules/ForbiddenNestedForeachWithEmptyStatementRule/Fixture/NotNestedForeach.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Tests\Rules\ForbiddenNestedForeachWithEmptyStatementRule\Fixture;
+
+foreach ($errors as $fileErrors) {
+
+}
```

Comment:
Could you add test case with nested stms, e.g. echo $fileErrors

Reply:
I added echo inside foreach.